### PR TITLE
Standardize iex bin files 

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: bin/iex [hunk 1]
```diff
@@ -15,11 +14,13 @@ if [ $# -gt 0 ] && ([ "$1" = "--help" ] || [ "$1" = "-h" ]); then
   --detached                  Starts the Erlang VM detached from console
   --erl SWITCHES              Switches to be passed down to Erlang (*)
   --hidden                    Makes a hidden node
+  --help, -h                  Prints this message and exits
```
Comment (josevalim) [inline bin/iex]: If this was meant to be alphabetic ordering, then it should be one up. :)

File: bin/iex.bat [hunk 2]
```diff
@@ -9,29 +9,30 @@ goto run
...
-echo  --name NAME                 Makes and assigns a name to the distributed node
-echo  --no-halt                   Does not halt the Erlang VM after execution
-echo  --sname NAME                Makes and assigns a short name to the distributed node
-echo  --werl                      Uses Erlang's Windows shell GUI (Windows only)
+echo   --app APP                   Starts the given app and its dependencies (*)
+echo   --cookie COOKIE             Sets a cookie for this distributed node
+echo   --detached                  Starts the Erlang VM detached from console
+echo   --erl SWITCHES              Switches to be passed down to Erlang (*)
+echo   --hidden                    Makes a hidden node
+echo   --help, -h                  Prints this message and exits
```
Comment (josevalim) [inline bin/iex.bat]: Same here.

File: bin/iex (hunk 1 above)
Comment (eksperimental) [inline bin/iex]: ooops.
fixed!

File: bin/iex.bat (hunk 2 above)
Comment (eksperimental) [inline bin/iex.bat]: fixed

**Comment by fishcakez** [comment]
Thank you :heart:

**Comment by eksperimental** [comment]
no worries @fishcakez 
Always a pleasure to contribute :-)

**Pull request merged**